testing.rst

```diff
@@ -211,6 +211,21 @@ Now you can use CSS selectors with the crawler. To assert that the phrase
         $crawler->filter('html:contains("Hello World")')->count()
     );
 
+.. versionadded:: 4.3
+
+    You can also use a shortcut:
```

Comment:
Please end with “::” to get php syntax highlighting

Reply:
Good catch, thanks 👍